validators/migrate-vali/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Migrate Validator"
+description: >
+  How to migrate an Autonity validator node to a new instance or to a new IP/Port address
+---
+
+::: {.callout-note title="Prerequisites" collapse="false"}
+- Ensure that the host machine meets the [minimum requirements](/node-operators/install-aut/#requirements)
+:::
+
+
+## Migrating validator node to a new environment
```

Comment:
I think it is a bit confusing that this part which is on top of the page links to the /node-operators/migrate-aut page which says you cannot change your IP.

I think people will get confused if they see this part first. I would move it down unless I am missing something here

Reply:
I've switched the order, and it looks better. I think you're right  on 'confusion'. 

(I'm wondering if it could be removed altogether from this page. That can be done in a follow on PR, if needs be.)

Commit: 02e11361535d5a6bd62ddc2d20100afbc89391d1.